llvm/docs/ReleaseNotes.rst

```diff
@@ -94,6 +94,11 @@ Changes to the AArch64 Backend
 
 * Added support for Cortex-A520, Cortex-A720 and Cortex-X4 CPUs.
 
+* Neoverse-N2 was incorrectly marked as an Armv8.5a core. This has been
+  changed to an Armv9.0a core. However, crypto options are not enabled
+  by default for Armv9 cores, so `-mcpu=neoverse-n2+crypto` is required
```

Comment:
`is now required` ?

Reply:
It's my understanding that v8.5-a == v9.0-a as far as the compiler is concerned (there are a couple of extra requirements to be v9.0-a that don't concern the compiler). Is that correct?

Might be worth stating that they enable the same features, with the exception of crypto as mentioned.
(not saying that they are literally the same, just same as far as a compiler is concerned)

Reply:
The SVE2 family of features seems to be v9.0-a only. They are not available in v8.5-a.

Reply:
That's true but my concern as a user would be does v9.0-a add anything implicitly that wasn't there before. I don't think it does here but worth noting just to save the user the effort of checking.

Reply:
Also clang would let you add sve2 to almost anything, but that's because it doesn't model the rules at all :)

Reply:
After this PR the N2 secretly got support for SVE2. v8.1-a+sve2 is a ton of problems.

Reply:
That is a problem worth fixing but it's not unique to this change unfortunately. Examples like this succeed:
```
$ ./bin/clang -march=armv8.1-a+sve2 /tmp/test.c
```
Which is a larger issue with clang.

At least with this change, `-mcpu=neoverse-n2+sve+sve2` follows the architecture rules where previously it did not. So that's a good improvement.

Reply:
Thanks for your comments. I've adjusted the wording in another commit to:

```
 * Neoverse-N2 was incorrectly marked as an Armv8.5a core. This has been
   changed to an Armv9.0a core. However, crypto options are not enabled
   by default for Armv9 cores, so `-mcpu=neoverse-n2+crypto` is now required
   to enable crypto for this core. As far as the compiler is concerned,
   Armv9.0a has the same features enabled as Armv8.5a, with the exception
   of crypto.
```